packages/context-editor/src/components/MenuBar.tsx

```diff
@@ -208,21 +210,30 @@ const paragraphTypeItems: MenuItem[] = [
 ];
 
 const ParagraphDropdown = () => {
-	const { view } = usePluginViewContext();
-	const activeType = paragraphTypeItems.find((item) => item.command(view)(view.state).isActive);
+	const itemCommand = useEditorEventCallback((view, item: MenuItem) => {
```

Comment:
rather than change the structure of the commands, I'm wrapping them in the `useEditorEventCallback` so that they have access to the `view`